src/metric_config.cpp

```diff
@@ -0,0 +1,181 @@
+    }
+
+    if (v.HasMember("endpoint_path")) {
+        endpointsPath = v["endpoint_path"].GetString();
+    } else {
+        endpointsPath = "/metrics";
+    }
+
+    if (v.HasMember("metrics_list")) {
+        status = parseMetricsArray(v["metrics_list"]);
+    } else {
+        setAllMetricsTo(metricsEnabled);
+    }
+
+    return status;
+}
+
+Status MetricConfig::parseMetricsArray(const rapidjson::Value& v) {
+    for (auto& sh : v.GetArray()) {
+        std::string metric = std::string(sh.GetString());
+        if (metric == "requestSuccessGrpcPredict") {
+            requestSuccessGrpcPredict = true;
+        }
+        if (metric == "requestSuccessGrpcGetModelMetadata") {
+            requestSuccessGrpcGetModelMetadata = true;
+        }
+        if (metric == "requestSuccessGrpcGetModelStatus") {
+            requestSuccessGrpcGetModelStatus = true;
+        }
+        if (metric == "requestSuccessRestPredict") {
+            requestSuccessRestPredict = true;
+        }
+        if (metric == "requestSuccessRestGetModelMetadata") {
+            requestSuccessRestGetModelMetadata = true;
+        }
+        if (metric == "requestSuccessRestGetModelStatus") {
+            requestSuccessRestGetModelStatus = true;
+        }
+        if (metric == "requestFailGrpcPredict") {
+            requestFailGrpcPredict = true;
+        }
+        if (metric == "requestFailGrpcGetModelMetadata") {
+            requestFailGrpcGetModelMetadata = true;
+        }
+        if (metric == "requestFailGrpcGetModelStatus") {
+            requestFailGrpcGetModelStatus = true;
+        }
+        if (metric == "requestFailRestPredict") {
+            requestFailRestPredict = true;
+        }
+        if (metric == "requestFailRestGetModelMetadata") {
+            requestFailRestGetModelMetadata = true;
+        }
+        if (metric == "requestFailRestGetModelStatus") {
+            requestFailRestGetModelStatus = true;
+        }
+        if (metric == "requestSuccessGrpcModelInfer") {
+            requestSuccessGrpcModelInfer = true;
+        }
+        // KFS
+        if (metric == "requestSuccessGrpcModelMetadata") {
+            requestSuccessGrpcModelMetadata = true;
+        }
+        if (metric == "requestSuccessGrpcModelStatus") {
+            requestSuccessGrpcModelStatus = true;
+        }
+        if (metric == "requestSuccessRestModelInfer") {
+            requestSuccessRestModelInfer = true;
+        }
+        if (metric == "requestSuccessRestModelMetadata") {
+            requestSuccessRestModelMetadata = true;
+        }
+        if (metric == "requestSuccessRestModelStatus") {
+            requestSuccessRestModelStatus = true;
+        }
+        if (metric == "requestFailGrpcModelInfer") {
+            requestFailGrpcModelInfer = true;
+        }
+        if (metric == "requestFailGrpcModelMetadata") {
+            requestFailGrpcModelMetadata = true;
+        }
+        if (metric == "requestFailGrpcModelStatus") {
```

Comment:
There is no "ModelStatus" in KFS. We should rename it to "ModelReady"

Reply:
Done.